src/Resources/app/administration/src/module/sw-in-app-purchases/component/sw-in-app-purchase-checkout/index.ts

```diff
@@ -52,56 +52,76 @@ export default Shopware.Component.wrapComponentConfig({
     },
 
     methods: {
-        createdComponent() {
-            this.store.$subscribe(() => {
-                this.requestFeature();
-            });
-        },
-
-        async requestFeature() {
+        requestFeature() {
             if (!this.store.extension || !this.store.entry) {
                 this.reset();
-
                 return;
             }
 
             this.state = 'loading';
 
-            await Promise.all([
+            Promise.all([
```

Comment:
if you're not awaiting anymore, this function not waits for the promise to finish before returning
```suggestion
            await Promise.all([
```